### ISSUE TITLE
Can Kadira package still be used ? 

### ISSUE THREAD

**Issue by tomvolek** [meteorhacks/kadira#263]
Can Kadira package still be used for performance testing, considering kadira.io is no longer around ? 

Tom

**Comment by kodie** [comment]
I wrote an article about how I got Kadira up and running on a Digital Ocean droplet here: https://www.gethappyboards.com/2017/07/rolling-out-your-own-instance-of-kadira/

**Comment by GbalsaC** [comment]
@kodie Excellent read! One tip on the nodejs var, if you install nvm it will setup binaries for the selected node version and everything will work using node instead of linking to nodejs (which I believe it's not always updated)

**Comment by Brouilles** [comment]
@kodie I follow your blog post, but the following command `mongo mongodb://kadira:meteor@candidate.51.mongolayer.com:10113,candidate.50.mongolayer.com:10369/kadira-data?replicaSet=set-55e152292e37af9f6130129a` return an error. 

```
Error: connect failed to replica set set-55e152292e37af9f6130129a/candidate.51.mongolayer.com:10113,candidate.50.mongolayer.com:10369 :
connect@src/mongo/shell/mongo.js:240:13
@(connect):1:6
exception: connect failed
```

**Comment by abernix** [comment]
It's worth pointing out in this thread (which seems to come up on Google searches) that the `meteorhacks:kadira` package is no longer maintained in this repository (see #264).

This package lives on as `mdg:meteor-apm-agent`, and should be mostly compatible with this (old) package.

**Comment by ilan-schemoul** [comment]
Installing seem to be quite complex, if I calculated right  as you go galaxy servers are 57.6$ a months versus 5$ a month for digital ocean, that's too expensive for me. Is there simple and free monitoring tool for Meteor ?

**Comment by po-jo** [comment]
Yes, I'm using Monti APM (based on Kadira). So far it's convenient :)
